libs/server/Storage/Functions/ObjectStore/ReadMethods.cs

```diff
@@ -58,5 +39,9 @@ public bool ConcurrentReader(ref byte[] key, ref SpanByte input, ref IGarnetObje
             dst.garnetObject = value;
             return true;
         }
+
+        /// <inheritdoc />
+        public bool ConcurrentReader(ref byte[] key, ref SpanByte input, ref IGarnetObject value, ref GarnetObjectStoreOutput dst, ref ReadInfo readInfo, ref RecordInfo recordInfo)
```

Comment:
Why not consolidate to one single method if they're both doing the same thing?

Reply:
It is consolidated into one method (SingleReader). ConcurrentReader calls into SingleReader now.

Reply:
Tsavorite calls into these under different circumstances, so they both need to be in here. They are part of the interface.